core/templates/dev/head/css/oppia.css

```diff
@@ -2077,16 +2161,11 @@ div#ng-curtain {
   font-weight: normal;
 }
 
-.oppia-right-editor-gadget-panel,
-.oppia-left-editor-gadget-panel,
 .oppia-bottom-editor-gadget-panel {
   border: 1px solid silver;
```

Comment:
Updated and added TODO(anuzis): Remove bottom-editor-gadget-panel in UI update (working on this later today).